model_templates/training/python3_sklearn_transform_no_hook/README.md

```diff
@@ -0,0 +1,45 @@
+## Sklearn Transform Template
```

Comment:
I ended up adding a bunch of almost-identical model templates to cover test cases, which seems pretty lame. I'm sure there's a better way

Reply:
do we use r2d2 in tests?

Reply:
I don't know- who would know?

Reply:
@yakov-g

Reply:
We don't use r2d2 in tests because it's just a tool to test users environment.

@jlpolit 
Can you just do these tests like we do in integration tests: code dir is assembled for every test case: with custom/ without custom, etc.

Reply:
good call - will update accordingly